### ISSUE TITLE
news - alternate featured image position

### ISSUE THREAD

**Issue by Jimmi08** [Jimmi08/e107_snippets#9]
To get something like this in news listing:

![image](https://cloud.githubusercontent.com/assets/5429548/15896197/8747c774-2d8f-11e6-9811-c385f38c69c9.png)

```
	function sc_news_right_image() {
	  global $pairing;
	  $data = e107::getScBatch('news')->getScVar('news_item');
	  if($pairing)   
		{ 
		  $text = 'right-img';
			$pairing = false;
			return $text;
		}
		else {
			$pairing = true;
			return '';
		
		};
	}
```

in template:

```
$NEWS_TEMPLATE['list']['item']	= ' 
{SETIMAGE: w=600&h=400}
<article class="blogcontainer {NEWS_RIGHT_IMAGE}">
...
</article>';
```

**Comment by rica-carv** [comment]
If i'm allowed, just some remarks:
Why you need this line, if $data is not used???
`$data =e107::getScBatch('news')->getScVar('news_item');`

I would get rid of the $text var from here, since it's only need once: `$text = 'right-img';
            $pairing = false;
            return $text;`
Like this:
`$pairing = false;
            return 'right-img';`

On final note i'm almost tempeted in find a way to make all that` if else `sentence in a single line..... :laughing:

**Comment by Jimmi08** [comment]
Good catch, because copying :) You are right, of course.  I focused on way how to remove global variable (not founded), that I missed other things.  With if/else it's my oldschool roots, but after confirming that this way it works, I can use better way - I always forget how it is called - tri... 
If you write that code, I will be glad. Thank you.

**Comment by rica-carv** [comment]
Now on a cooldown thinking, maybe single line won't work (ternary operator, lol)....
All because you're assigning a variable, and do the return the same way....
Anyway, here's my best approach (untested):
`global $pairing;
$pairing = !$pairing;
return ($pairing ? 'right-img' : "")`

**Comment by Jimmi08** [comment]
Waw. Thank you,

**Comment by rica-carv** [comment]
P.S.: I'm a newbie php coder.... lol

**Comment by Jimmi08** [comment]
Me too, I am used to different language (4GL), I probably never get used to work with ! or ternary operator.

**Comment by rica-carv** [comment]
Well, i started with [Basic](https://en.wikipedia.org/wiki/Sinclair_BASIC), moved to [GAC](https://en.wikipedia.org/wiki/Graphic_Adventure_Creator), and aftewards was forced to learn [AutoLISP](https://en.wikipedia.org/wiki/AutoLISP), and [VBA](https://en.wikipedia.org/wiki/Visual_Basic_for_Applications).... a real programming languages mess.... LOL
